src/resources/locales/de.json

```diff
@@ -151,6 +151,10 @@
             "bookmarkTitle": "bookmark",
             "detachWindowTitle": "das Fenster lösen",
             "fullscreenTitle": "fullscreen mode",
+            "goTo": "Go to",
+            "goToError": "This page does not exist",
+            "goToPlaceHolder": "Go to page",
+            "goToTitle": "Change page",
```

Comment:
Anybody with German skills? :)
@panaC ? @llemeurfr ?

Reply:
I have my friend https://www.deepl.com/translator , if you likes, :)

Reply:
I translated it with google translate